doc/engines.rst

```diff
@@ -131,8 +131,9 @@ The general operation of the DSP engines is illustrated in the diagram below:
    \draw[queue] (group) -- (upload);
    \end{scope}
 
-The F-engine uses two input streams and aligns two incoming polarisations, but
-in the XB-engine there is only one.
+The F-engine and V-engine each use two input streams to align two incoming
+polarisations, but in the XB-engine there is only one as the polarisations are
+already interleaved in the incoming packets.
```

Comment:
Very minor observation. I reckon these can be two separate sentences. The second sentence can read something like
> The XB-engine only uses one input stream as the polarisations are already interleaved in the incoming packets.

Just breaks up any ambiguity/confusion (like I had when first reading the sentence).